js/blocks/controls/color-picker.js

```diff
@@ -0,0 +1,68 @@
+import Color from 'color';
+
+const { __ } = wp.i18n;
+const { BaseControl, ColorPicker, Dropdown, Tooltip } = wp.components;
+
+const BlockLabColorPickerControl = ( props, field, block ) => {
+	const { setAttributes } = props;
+	const attr = { ...props.attributes };
+	const customColorPickerLabel = __( 'Custom color picker' );
+
+	// Set a default color of White, if nothing is set.
+	attr[ field.name ] = attr[ field.name ] ? attr[ field.name ] : '#fff';
+
+	const currentColor = {
+		display: 'inline-block',
+		width: '120px',
+		height: '30px',
+		borderRadius: '5px',
+	}
+
+	return (
+		<BaseControl
+			label={ field.label }
+			help={ field.help }
+			className="block-lab-color-picker"
+		>
+			<span className="components-color-picker__alpha block-lab-color-picker__current-color" style={ {
```

Comment:
Could this be replaced with the `ColorIndicator` component? https://github.com/WordPress/gutenberg/tree/master/packages/components/src/color-indicator

Reply:
It could, but would require some adjusting non-the-less. Do you see any advantages of using the component over the current implementation?

Reply:
Relying on core components means less maintainability.

Reply:
While I do agree, in this case we are only referencing a span and, even if we did use the component, we still need the wrapper component and style adjustments (which the core component does not allow to alter on the component level).